main.c

```diff
@@ -41,8 +41,10 @@ static struct fuse_operations kibosh_oper;
 static const char * const MANDATORY_FUSE_OPTIONS[] = {
     "-oallow_other", // Allow all users to access the mount point.
     "-odefault_permissions", // tell FUSE to do permission checking for us based on the reported permissions
-    "-odirect_io", // Don't cache data in FUSE.
     "-ohard_remove", // Do not translate unlink into renames to .fuse_hiddenXXX
+    "-odirect_io",
```

Comment:
let's just remove direct_io and leave the rest the same now.  thx

Reply:
Thanks for the review.
direct_io was removed from the options.
default_permissions was kept.
chown was added to kibosh_mkdir and kibosh_open_normal_file_impl to change the owner of new directory/file to the actual user.